.github/workflows/release.yml

```diff
@@ -34,16 +40,33 @@ jobs:
   deploy-docker:
     runs-on: ubuntu-latest
 
+    if: ${{ always() }}
     needs: deploy-pypi
     steps:
       - uses: actions/checkout@v2
-      - name: Login
-        env:
-          DOCKER_USER: ${{secrets.DEVOPSHQ_DOCKER_USER}}
-          DOCKER_PASSWORD: ${{secrets.DEVOPSHQ_DOCKER_TOKEN}}
-        run: |
-          docker login -u $DOCKER_USER -p $DOCKER_PASSWORD
-      - name: Build
-        run: docker build . --file docker/Dockerfile --tag devopshq/artifactory-cleanup:latest
-      - name: Push
-        run: docker push devopshq/artifactory-cleanup:latest
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: devopshq/artifactory-cleanup
+          flavor: latest=true
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+
+      - name: Login to image repository
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DEVOPSHQ_DOCKER_USER }}
+          password: ${{ secrets.DEVOPSHQ_DOCKER_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: docker/Dockerfile
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Could you add a short "RELEASE" section to README.md? 

Now we need to do two steps:
1. Bump version in https://github.com/devopshq/artifactory-cleanup/blob/master/setup.py
2. Create a release (or push a tag, but let's use Github for it)